src/daemon/config.rs

```diff
@@ -408,8 +408,8 @@ impl Config {
         uri::Https::from_str(&self.service_uri).unwrap()
     }
 
-    pub fn ta_cert_uri(&self) -> uri::Https {
-        uri::Https::from_string(format!("{}ta/ta.cer", &self.service_uri)).unwrap()
+    pub fn rfc8181_uri(&self, publisher: &PublisherHandle) -> uri::Https {
```

Comment:
Is this a breaking change?

Reply:
yes, this is now in the [testbed] config.